### PR TITLE
first commit, everything is working and folks get disabled/deleted properly

### PR DESCRIPTION
**Pull request by JohnTheodore**
... Will get setup docker / openshift next.

### CODE REVIEW

**Review by bdastur — APPROVED** [review]
Changes look good
👍 👍

File: aws_ad_accounts_sync.py [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+    except Exception as error:
+      logger.exception(error)
+      # if we regularly have exceptions, let aws slack know about it once per day.
+      if error_counter == 0:
+        slack_error = '```%s```' % error
+        message_slack(slack_error)
+      elif 47 > error_counter > 0:
+        error_counter += 1
+      else:
+        error_counter = 0
```
Comment (cviecco) [inline aws_ad_accounts_sync.py]: The error counter should not be reset here.. it should get reset as the last statement of the try.

File: aws_ad_accounts_sync.py [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+  logging.getLogger('requests').setLevel(logging.ERROR)
+  logging.getLogger('botocore').setLevel(logging.ERROR)
+  error_counter = 0
+  while True:
+    try:
+      sync_aws_ad()
+    except Exception as error:
+      logger.exception(error)
+      # if we regularly have exceptions, let aws slack know about it once per day.
+      if error_counter == 0:
```
Comment (cviecco) [inline aws_ad_accounts_sync.py]: error_counter always increments...
if (error_counter%48) == 4   {
   /// only send slack on the 4th successive failure .. notify again  in 24 hours time.
}

**Comment by cviecco** [comment]
LGTM

**Pull request merged**